pkg/placement/placement.go

```diff
@@ -207,7 +188,12 @@ func (p *Service) Run(ctx context.Context, listenAddress, port string) error {
 	if err != nil {
 		return fmt.Errorf("failed to listen: %w", err)
 	}
-	grpcServer := grpc.NewServer(sec.GRPCServerOptionMTLS())
+
+	keepaliveParams := keepalive.ServerParameters{
+		Time:    1 * time.Second,
```

Comment:
We can discuss these numbers

Reply:
:+1: 1 second is fairly aggressive, but not insane. gRPC library uses 5 seconds but default https://github.com/grpc/grpc-go/blob/e22436abb80982dab3294250b7284eedd8f20768/examples/features/keepalive/server/main.go#L47

Reply:
We've seen recently a few examples of managed K8s clusters running in what appeared to be sporadically unreliable networks. In these cases 1 second does seem a bit aggressive, however if we're confident that reconnections shouldn't cause any widespread issues then I guess its fine to keep

Reply:
`Time` in this case is the period of inactivity after which the server will try to ping the client to check if the stream is still alive. Knowing that the sidecars pings every second, maybe we can relax this requirement, and make it 2 seconds. 
`Timeout` represents the duration after the ping during which the server will wait for a response, after which the connection is closed. I think we can increase this to 3 seconds, which will match the old faulty host detection timeout of 3 seconds.